gui/default/syncthing/transfer/localChangedFilesModalView.html

```diff
@@ -1,8 +1,12 @@
-<modal id="localChanged" status="info" icon="fas fa-exclamation-circle" heading="{{'Locally Changed Items' | translate}}" large="yes" closeable="yes">
-  <div class="modal-body">
-    <p translate>
+<modal id="localChanged" status="{{localChangedType === 'receiveencrypted' ? 'warning' : 'info'}}" icon="fas fa-exclamation-circle" heading="{{localChangedHeading(localChangedType)}}" large="yes" closeable="yes">
+  <div class="modal-body" ng-switch="localChangedType">
+    <p ng-switch-when="receiveonly" translate>
       The following items were changed locally.
     </p>
+    <p ng-switch-when="receiveencrypted">
+      <span translate>The following items are present and not encrypted.</span>
```

Comment:
So we don't know if they are encrypted or not, but are unexpected as they were not received.

Reply:
Any ideas for a understandable/readable sentence that means something? As opposed to what's there now or something like "The following unexpected items are present in the folder."

Reply:
`The following items are present in the folder but were not received from remote devices`

or what you had makes sense too.